packages/components/src/radio/src/Radio.css

```diff
@@ -2,45 +2,46 @@
     display: inline-flex;
     align-items: center;
     cursor: pointer;
-    line-height: var(--o-ui-lh-1);
+    line-height: var(--hop-body-md-line-height);
     max-width: 100%;
     width: max-content;
     height: max-content;
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
 }
 
 /* BUTTON */
 .o-ui-radio-button {
     display: inline-block;
-    background-color: var(--o-ui-bg-alias-surface);
-    border-radius: var(--o-ui-br-circular);
-    border: 1px solid var(--o-ui-b-alias-mid-break);
+    background-color: var(--hop-neutral-surface);
+    border-radius: var(--hop-shape-circle);
+    border: 1px solid var(--hop-neutral-border);
     position: relative;
-    width: var(--o-ui-sz-3);
-    height: var(--o-ui-sz-3);
+    width: 1.5rem;
+    height: 1.5rem;
     flex-shrink: 0;
 }
 
 .o-ui-radio-button::before {
+    --_radio-button-size: 0.75rem;
     content: "";
     display: block;
-    background-color: var(--o-ui-bg-alias-accent);
-    border-radius: var(--o-ui-br-circular);
+    background-color: var(--hop-neutral-icon-active);
+    border-radius: var(--hop-shape-circle);
     transform: scale(0);
     transition: all var(--o-ui-easing-duration-1) var(--o-ui-easing-productive);
     position: absolute;
     top: 50%;
     left: 50%;
-    width: var(--o-ui-sz-3);
-    height: var(--o-ui-sz-3);
-    margin-top: calc(var(--o-ui-sz-3) * -1 / 2);
-    margin-left: calc(var(--o-ui-sz-3) * -1 / 2);
+    width: var(--_radio-button-size);
+    height: var(--_radio-button-size);
+    margin-top: calc(var(--_radio-button-size) * -1 / 2);
+    margin-left: calc(var(--_radio-button-size) * -1 / 2);
 }
 
 /* CONTENT | LABEL */
 .o-ui-radio-label {
-    line-height: var(--o-ui-lh-2);
-    margin-left: var(--o-ui-sp-2);
+    line-height: var(--hop-body-md-line-height);
```

Comment:
Is it still necessary??

Reply:
If we remove the one on line 5, yes, it will be necessary.

Reply:
I think he means it's not necessary because it's already set elsewhere for .o-ui-text-md:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/86a32552-3d1b-4f87-812c-8f0a46ca6d7a)